src/locker.rs

```diff
@@ -47,15 +42,8 @@ impl<'a> Drop for Locker<'a> {
   }
 }
 
-impl<'a> Deref for Locker<'a> {
-  type Target = LockedIsolate;
-  fn deref(&self) -> &LockedIsolate {
-    self.isolate
-  }
-}
-
-impl<'a> DerefMut for Locker<'a> {
-  fn deref_mut(&mut self) -> &mut LockedIsolate {
-    self.isolate
+impl<'a> LockedIsolate for Locker<'a> {
+  fn isolate(&mut self) -> &mut CxxIsolate {
+    unsafe { std::mem::transmute_copy(&self.isolate) }
```

Comment:
😮

Reply:
This transmute is actually not necessary any more. Will remove it tomorrow.